src/tools/fuzzing.h

```diff
@@ -2640,6 +2642,13 @@ class TranslateToFuzzReader {
     return builder.makeRefIsNull(make(getReferenceType()));
   }
 
+  Expression* makeRefEq(Type type) {
+    assert(type == Type::i32);
+    assert(wasm.features.hasReferenceTypes() && wasm.features.hasGC());
+    return builder.makeRefEq(make(getEqReferenceType()),
+                             make(getEqReferenceType()));
```

Comment:
The reason for the CI failure is that the evaluation order of these arguments is undefined.

Reply:
Thanks, would never have thought about that. In fact, just learned that this is the case. 😄 that's simultaneously amazing and deeply concerning for both the language at hand and the status of my obviously still superficial knowledge.

Reply:
Yep, C/C++ evaluation order is pretty bonkers.

Reply:
The lack of order is surprising, yeah, but it's an opportunity for the optimizer...

Reply:
Feeling tempted to make a "but it's an opportunity for the optimizer" meme, showing a person with their computer on fire 😄